js/create-bookmarks-folder-from-window.js

```diff
@@ -0,0 +1,19 @@
+import { getAllTabsInCurrentWindow } from "./utils.js";
+
+async function createBookmarkFromWindow() {
+  const allTabsInWindow = await getAllTabsInCurrentWindow();
+
+  // Create a bookmarks folder with the tabs
+  chrome.bookmarks.create({ 'title': 'My Tabs Folder', 'index': 0 }, function (newFolder) {
```

Comment:
Separate out the fields that need to be passed into bookmarks.create into separate lines. Looks cleaner.

// Create a bookmarks folder with the tabs
  chrome.bookmarks.create({
    'title': 'My Tabs Folder', 
    'parentId': '1' 
  }, function (newFolder) {
    allTabsInWindow.forEach(function (tab) {
      chrome.bookmarks.create({
        'parentId': newFolder.id, 
        'title': tab.title, 
        'url': tab.url 
      });
    });
  });